CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Any change to this repository is always welcomed, and greatly appreciated. You can add new entries, or modify/remove existing entries if they are changed/discontinued. You are also allowed to add/modify categories as you see fit.
```

Comment:
I would reword this first sentence and remove information that is redundant.

"Contributions to the list are greatly appreciated and always welcome. Feel free to add and reorganize items as you see fit for a Pull Request"

Though that being said, I think we should avoid giving people the impression that they should reorganize the entire list with each PR.

Reply:
Maybe something like:

> Contributions to the list are greatly appreciated and always welcome. Feel free to add and reorganize items and categories as you see fit. Although, it is not necessary to reorganize every item with every Pull Request.

Reply:
I think just the first two sentences are good enough.